elements/pfe-dropdown/src/pfe-dropdown-item.js

```diff
@@ -0,0 +1,79 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeDropdownItem extends PFElement {
+  static get tag() {
+    return "pfe-dropdown-item";
+  }
+
+  get templateUrl() {
+    return "pfe-dropdown-item.html";
+  }
+
+  get styleUrl() {
+    return "pfe-dropdown-item.scss";
+  }
+
+  static get observedAttributes() {
+    return ["pfe-type", "disabled"];
```

Comment:
We should change the `pfe-type` attribute. We're already using this for determining if the component is a container, content, or a combo. What if we changed this attribute name to `pfe-item-type`?